wagtail/snippets/templates/wagtailsnippets/snippets/type_index.html

```diff
@@ -1,4 +1,4 @@
-{% extends "wagtailadmin/base.html" %}
+{% extends "wagtailadmin/generic/index.html" %}
```

Comment:
It might look a bit risky to do this, but it's actually safer than it looks.

The `generic/index.html` overrides the following blocks:
- `titletag`
- `content`
- `extra_js`
- `extra_css`

The first two blocks are overridden by `type_index.html` without any `{{ block.super }}` calls. Meanwhile, the `extra_js` and `extra_css` blocks include `{{ media.js }}` and `{{ media.css }}`, which are what we need.

Hence, the only changes to this template are to extend from `generic/index.html` and make use of `{{ block.super }}` in the `extra_js` block.